apps/mobile/src/components/cell.tsx

```diff
@@ -0,0 +1,75 @@
+import { ResponsiveValue, useTheme } from '@shopify/restyle';
```

Comment:
This is the `Cell` from [here](https://github.com/leather-io/issues/issues/229) right? 

Maybe we can just add it to the UI lib from the start?

Reply:
I did some initial work on this in https://github.com/leather-io/mono/pull/398 

I can finish moving it once this is merged

Reply:
The `Cell` should be comparable to the `Pressable` component in the extension, it should often be just a wrapper arnd `ItemLayout`?

Reply:
OK thanks @fbwoolf . Maybe I will should rename `Pressable` as `Cell`?

Reply:
I am pretty open to naming and have a hard time tracking where the names come from, but seems to make sense to have them coordinated if the same in web/mobile. `Pressable` prob isn't a great name for web, and RN already has a `Pressable` component.

Reply:
I thought `Cell` was an Apple term from https://developer.apple.com/design/human-interface-guidelines/ but it's not. 

Let's just call it `Cell` so that we can group the components together. I honestly don't mind what we call them but `Cell` could be an upgrade on `Pressable` if it avoids clashing with RN `Pressable` also